Rubberduck.Parsing/Inspections/Resources/InspectionsUI.de.resx

```diff
@@ -876,4 +855,16 @@ Falls der Parameter 'null' sein kann, bitte dieses Auftreten ignorieren. 'null'
   <data name="StopKeywordInspectionName">
     <value>Stop keyword</value>
   </data>
-</root>
+  <data name="ReplaceIfElseWithConditionalStatementQuickFix" xml:space="preserve">
+    <value>Ersetze If/Else mit einer einfachen Zuordnung </value>
+  </data>
+  <data name="BooleanAssignedInIfElseInspectionResultFormat" xml:space="preserve">
+    <value>Boolean Ausdruck '{0}' wurde optional zugewiesen </value>
+  </data>
+  <data name="BooleanAssignedInIfElseInspectionName" xml:space="preserve">
+    <value>Boolean Ausdruck als optionale Zuordnung</value>
+  </data>
+  <data name="BooleanAssignedInIfElseInspectionMeta" xml:space="preserve">
+    <value>Ein Element wurde mit True/False in verschiedenen Zweigen einer If-Bedinung ohne weitere optionale Bedingungen zu gewiesen. Benutze statt dessen die Bedingung direkt mit dem Element. </value>
```

Comment:
typo: "If-Bedinung".

Also the sentence structure is a bit difficult to follow. Try getting more of the meaning and less of the syntax into translation

Reply:
two more typos: 
"zu gewiesen" --> "zugewiesen"
"statt dessen" --> "stattdessen"